SampleService.spec

```diff
@@ -508,7 +508,15 @@ module SampleService {
     } ValidateSamplesParams;
 
     typedef structure {
-        mapping<sample_name, list<string>> errors;
+        string message;
+        string dev_message;
+        sample_name sample;
```

Comment:
I think having this field called sample may be a bit confusing, `sample_name` might be clearer.  not a big deal, but may help to be consistent here